ethylene-yaml/build.gradle

```diff
@@ -3,7 +3,7 @@ plugins {
 }
 
 group 'com.github.steanky'
-version '0.1.0'
+version '0.3.0'
```

Comment:
kind of a side note but why have a version inconsistency

Reply:
cuz i added ethylene-yaml and ethylene-hjson when the other packages were already on version 0.2.0 so they've been out of sync since

Reply:
i'll keep them synchronized for 1.0.0 release lol